### PR TITLE
fix: use session storage to save user's selection

### PR DESCRIPTION
**Pull request by edewit**
fixes: #915

### CODE REVIEW

File: packages/launcher-component/src/flows/create-new-app-flow.tsx [hunk 1]
```diff
@@ -25,13 +25,25 @@ const defaultCustomApp = {
...
   },
   deployment: defaultDeploymentFormValue,
 };
 
 export function CreateNewAppFlow(props: { onCancel?: () => void }) {
-  const [app, setApp] = useState<CustomApp>(defaultCustomApp);
+  const [_app, _setApp] = useLocalStorage('app', JSON.stringify(defaultCustomApp));
+  const app: CustomApp = JSON.parse(_app);
+  const setApp = (value: CustomApp) => {
+    _setApp(JSON.stringify(value));
```
Comment (ia3andy) [inline packages/launcher-component/src/flows/create-new-app-flow.tsx]: @edewit isn't it possible to make the `JSON.stringify` and `JSON.parse` part of the `useLocalStorage`?

File: packages/launcher-component/package.json [hunk 2]
```diff
@@ -14,7 +14,8 @@
   },
   "dependencies": {
     "launcher-client": "*",
-    "project-name-generator": "2.1.6"
+    "project-name-generator": "2.1.6",
+    "react-use-sessionstorage": "^1.0.0"
```
Comment (gastaldi) [inline packages/launcher-component/package.json]: Better use the exact version?

**Comment by ia3andy** [comment]
It's great you can refresh now without loosing the state :)

@edewit Cancel (in the different flows) is not working anymore

**Comment by ia3andy** [comment]
@edewit ok I fixed the cancel ;)

**Comment by ia3andy** [comment]
@gastaldi what is the workflow check?

**Comment by gastaldi** [comment]
@ia3andy It is the CircleCI build

**Comment by gastaldi** [comment]
There are conflicts in this PR

**Pull request merged**